adam-core/src/test/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDDSuite.scala

```diff
@@ -22,6 +22,15 @@ import org.bdgenomics.adam.util.ADAMFunSuite
 
 class GenotypeRDDSuite extends ADAMFunSuite {
 
+  sparkTest("union two genotype rdds together") {
+    val genotype1 = sc.loadGenotypes(testFile("gvcf_dir/gvcf_multiallelic.g.vcf"))
+    val genotype2 = sc.loadGenotypes(testFile("small.vcf"))
+    val union = genotype1.union(genotype2)
+    assert(union.rdd.count === (genotype1.rdd.count + genotype2.rdd.count))
+    assert(union.sequences.size === (genotype1.sequences.size + genotype2.sequences.size))
```

Comment:
This case is everyone's jolly favorite bioinformatics boondoggle: `chr` prefixes vs no prefixes. Thus, the contigs are distinct.